pkg/fixtures/fixtures.go

```diff
@@ -98,6 +99,13 @@ func NewFixtureFromFile(fs afero.Fs, apiKey, stripeAccount, baseURL, file string
 		}
 	}
 
+	if edit {
+		filedata, err = fxt.Edit(file, filedata)
+		if err != nil {
+			return nil, err
+		}
+	}
+
```

Comment:
do we still want to allow other edit flags (ie `override` `add` `remove` `skip`) if we are using `edit` flag?

Reply:
good question! I couldn't think of a really strong reason to **block** users from doing this (aside from the fact that it makes it slightly easier to mess yourself up, like by editing the fixture first and then trying to override a value that isn't there)

But I think that the scenario where devs use `--edit` to change just one value one-off in their fixture, while also having a long list of `--add`s (more convenient if you're adding something every single time) outweighed that possibility so I decided to keep it.

Maybe having the `edit` step go last makes more sense? Since you're visually seeing the fixture in that step, and might be confused why your `add`ed property isn't there

Reply:
i think its not that intuitive whether the `add/remove/skip/override` is applied before or after the `edit`. it might make sense to check that if user already has `edit` specified, then we dont want other fixture customization flags.

Reply:
fixed in https://github.com/stripe/stripe-cli/pull/992/commits/5423ce16a51fc0aa0c94de337c6f0cbe7d8798f2